map/map.js

```diff
@@ -184,6 +199,166 @@ var DefineMap = Construct.extend("DefineMap",{
 		return this;
 	},
 	/**
+	 * @function can-define/map/map.prototype.assignDeep assignDeep
+	 * @parent can-define/map/map.prototype
+	 *
+	 * @description Sets multiple properties on a map instance or a property that wasn't predefined.
```

Comment:
I am not entirely sure, I took it from the existing documentation on `map.set(propName, value)`

https://canjs.com/doc/can-define/map/map.prototype.set.html